### ISSUE TITLE
[wish-list] Can we have a gui tabline ??

### ISSUE THREAD

**Issue by skywind3000** [equalsraf/neovim-qt#305]
First of all, nvim-qt is a great project, I really love to switching from gvim to nvim + qt.
But missing gui tabline is the most important thing which stops my switching.

TUI tabline is too ugly on a modern qt ui. Can we have a gui version ??

**Comment by equalsraf** [comment]
Possible yes, nvim already exposes this feature over its API :D

**Comment by dimxdim** [comment]
Just to add my 2 cents, there are plenty of beautiful TUI tablines/bufferlines that also incorporate the colors of most popular colorschemes. Just look at airline, lightline, powerline etc.

There are also very good arguments against using tabs (and instead using buffers), as in 'vim tabs' have different semantics from say, web browsers' tabs. More here:
https://github.com/ap/vim-buftabline

Overall, it would be nice to have a GUI **bufferline** feature, but overall, I think not alot of people would use it, especially if it would not allow custom colors to match with colorscheme. One advantage of being "clickable" kinda beats the purpose, since the vast majority of users try to stay away from mouse as much as possible anyway (and for good reason). I could see some usefulness in rearranging buffer order by mouse drag-and-drop though.

Bottom line, IMO better focus on stability, speed and MS Windows support (there is no good terminal in windows, GUI Vim is the only realistic option now for Windows users and it leaves an impression of slowness).

**Comment by skywind3000** [comment]
gonvim is still buggy

**Comment by equalsraf** [comment]
This is already available, closing.

**Comment by skywind3000** [comment]
thanks
